internal/parser/parser.go

```diff
@@ -1085,35 +1145,32 @@ func (p *parser) parsePropertyList() []*ast.Property {
 
 func (p *parser) parseStringProperty() *ast.Property {
 	key := p.parseStringLiteral()
-	p.expect(token.COLON)
-	val := p.parsePropertyValue()
-	return &ast.Property{
-		Key:   key,
-		Value: val,
-		BaseNode: p.baseNode(p.sourceLocation(
-			locStart(key),
-			locEnd(val),
-		)),
-	}
+	return p.parsePropertySuffix(key)
 }
 
 func (p *parser) parseIdentProperty() *ast.Property {
 	key := p.parseIdentifier()
+	return p.parsePropertySuffix(key)
+}
 
-	var val ast.Expression
+func (p *parser) parsePropertySuffix(key ast.PropertyKey) *ast.Property {
+
```

Comment:
Can you remove these random extra lines that are added in various parts of the code? It's a nitpick, but they annoy me.